neonvm/runner/main.go

```diff
@@ -488,11 +488,13 @@ func main() {
 	var kernelPath string
 	var appendKernelCmdline string
 	var skipCgroupManagement bool
+	var diskCacheSettings string
 	flag.StringVar(&vmSpecDump, "vmspec", vmSpecDump, "Base64 encoded VirtualMachine json specification")
 	flag.StringVar(&vmStatusDump, "vmstatus", vmStatusDump, "Base64 encoded VirtualMachine json status")
 	flag.StringVar(&kernelPath, "kernelpath", defaultKernelPath, "Override path for kernel to use")
 	flag.StringVar(&appendKernelCmdline, "appendKernelCmdline", "", "Additional kernel command line arguments")
 	flag.BoolVar(&skipCgroupManagement, "skip-cgroup-management", false, "Don't try to manage CPU (use if running alongside container-mgr)")
+	flag.StringVar(&diskCacheSettings, "qemu-disk-cache-settings", "cache=none", "Cache settings to add to -drive args for VM disks")
```

Comment:
Mostly the idea was:

1. I want this to be easy to roll out region-by-region (otherwise would need to change CRD defaults, or cplane)
2. I don't think it _needs_ to be different for each VM; otherwise old VMs restarting would keep old cache settings, which imo is probably counterproductive

But yeah, storing this in the spec _would_ mean we'd only need to change the CRD and runner.

WDYT?

Reply:
Makes sense.